packages/botonic-react/src/device-config.js

```diff
@@ -0,0 +1,88 @@
+import {
+  getWebchatElement,
+  setWebchatElementHeight,
+  scrollToBottom,
+  getScrollableArea,
+} from './utils'
+
+export class DeviceConfig {
```

Comment:
if this does only keyboardresize, rename it accordingly, otherwise, split in different classes